lib/adapters/PgsqlAdapter.php

```diff
@@ -75,7 +75,7 @@ public function create_column(&$column)
 		$c->inflected_name	= Inflector::instance()->variablize($column['field']);
 		$c->name			= $column['field'];
 		$c->nullable		= ($column['not_nullable'] ? false : true);
-		$c->pk				= ($column['pk'] ? true : false);
+		$c->pk				= ($column['pk'] == 't');
```

Comment:
Are you sure about this change? I don't know postgres, so I can't be the judge of this..